style.css

```diff
@@ -0,0 +1,133 @@
+header {
```

Comment:
- Kindly, you font size, mergin, font weignt not the some, to the given figma design👍👍 please update your style based on the figma design

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/219629774-9faceb13-0e73-424b-92d4-dce28dbfc4fa.png" width="300">|<img src="https://user-images.githubusercontent.com/52010789/219629791-981f58af-26cf-48e2-a98e-4271720c2120.png" width="300">|


😎️

🤝️